_includes/manuals/2.1/1.2_release_notes.md

```diff
@@ -25,11 +25,13 @@ Foreman has been updated to run on the latest version of Ruby on Rails at the ti
 
 #### Support installation on EL8 distributions
 
-Foreman installation is now possible on EL8 distributions. The installer has been updated to support changes needed for these and yum repositories now contain packages for EL8. Note that installation is only supported and tested on RHEL and CentOS, packages may or may not work on Oracle Linux or Scientific Linux.
+Foreman installation is now possible on EL8 distributions. The installer has been updated to support changes needed for these and yum repositories now contain packages for EL8. Note that installation is only supported and tested on RHEL and CentOS, packages may or may not work on Oracle Linux.
```

Comment:
Can you please also update the quickstart and installation section to include instructions for el8?